server/middlewares/articlesMiddleware.js

```diff
@@ -31,38 +32,36 @@ export default class AriclesMiddleware {
       categoryId = '',
     } = req.body;
 
-    if (typeof title !== 'string') {
-      return Response.send(res, STATUS.BAD_REQUEST, [], 'title must be a string', false);
+
+    if (!title) {
```

Comment:
This would accept empty strings. All that needs to be in the input is '   ' and it will be accepted.